python/pyarrow/tests/test_gdb.py

```diff
@@ -39,13 +39,23 @@
 gdb_command = ["gdb", "--nx"]
 
 
+def environment_for_gdb():
+    env = {}
+    for var in ['PATH', 'LD_LIBRARY_PATH']:
```

Comment:
Thanks for this change. Was testing out on an MSYS2 environment, and the gdb tests wouldn't work without this.